src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/CaseProceedsInCasemanCallbackHandlerTest.java

```diff
@@ -170,5 +171,23 @@ void shouldNotAddPreviousCaseState_whenInvokedForLipVLipOrLrVLip() {
             assertThat(response.getData())
                     .extracting("previousCCDState").isNull();
         }
+
+        @Test
+        void shouldAddCoScApplicationStatusValue_whenInvoked() {
+            CaseData caseData = CaseData.builder()
+                .respondent1Represented(YesOrNo.NO)
+                .coSCApplicationStatus(CoscApplicationStatus.ACTIVE)
+                .build();
+
+            CallbackParams params = callbackParamsOf(caseData, ABOUT_TO_SUBMIT);
+            when(featureToggleService.isCoSCEnabled()).thenReturn(true);
+
+            AboutToStartOrSubmitCallbackResponse response =
+                (AboutToStartOrSubmitCallbackResponse) handler.handle(params);
+
+            assertThat(response.getData())
+                .extracting("coSCApplicationStatus")
+                .isEqualTo(CoscApplicationStatus.INACTIVE.toString());
+        }
```

Comment:
should we have another test to test the that the status shouldn't change if it's a different status thats not "Active"

Reply:
done